reports/README.md

```diff
@@ -0,0 +1,10 @@
+## Authoring krabby reports
```

Comment:
`docs` dir sounds like a great idea. I can move the two additional readmes to one now